distribution/docker/build.gradle

```diff
@@ -35,34 +39,42 @@ repositories {
     metadataSources { artifact() }
     content { includeGroup 'krallin' }
   }
+}
 
-  // Cloud builds bundle some beats
-  ivy {
-    name = 'beats'
-    if (useLocalArtifacts) {
-      url "file://${buildDir}/artifacts/"
-      patternLayout {
-        artifact '/[organisation]/[module]-[revision]-[classifier].[ext]'
-      }
-    } else {
-      url "https://artifacts-snapshot.elastic.co/"
-      patternLayout {
-        if (VersionProperties.isElasticsearchSnapshot()) {
-          artifact '/[organization]/[revision]/downloads/[organization]/[module]/[module]-[revision]-[classifier].[ext]'
-        } else {
-          // When building locally we always use snapshot artifacts even if passing `-Dbuild.snapshot=false`.
-          // Release builds are always done with a local repo.
-          artifact '/[organization]/[revision]-SNAPSHOT/downloads/[organization]/[module]/[module]-[revision]-SNAPSHOT-[classifier].[ext]'
+if(useDraWithBuildId == false) {
+  repositories {
+    // Cloud builds bundle some beats
+    ivy {
+      name = 'beats'
+      if (useLocalArtifacts) {
+        url "file://${buildDir}/artifacts/"
+        patternLayout {
+          artifact '/[organisation]/[module]-[revision]-[classifier].[ext]'
+        }
+      } else {
+        url "https://artifacts-snapshot.elastic.co/"
+        patternLayout {
+          if (VersionProperties.isElasticsearchSnapshot()) {
+            artifact '/[organization]/[revision]/downloads/[organization]/[module]/[module]-[revision]-[classifier].[ext]'
+          } else {
+            // When building locally we always use snapshot artifacts even if passing `-Dbuild.snapshot=false`.
+            // Release builds are always done with a local repo.
+            artifact '/[organization]/[revision]-SNAPSHOT/downloads/[organization]/[module]/[module]-[revision]-SNAPSHOT-[classifier].[ext]'
+          }
         }
       }
+      metadataSources { artifact() }
+      content { includeGroup 'beats' }
     }
-    metadataSources { artifact() }
-    content { includeGroup 'beats' }
   }
 }
 
 testFixtures.useFixture()
 
+repositories.all {
+  println it.name
```

Comment:
Let's make sure this doesn't accidentally sneak in.

Reply:
removed